src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -144,7 +144,9 @@ export default defineConfig({
 });
 ```
 
-Configuring `edgeMiddleware: true` will deploy your middleware as an Edge Function, and run it on all routes - including pre-rendered pages. However, locals specified in the middleware won't be available to any pre-rendered pages, because they've already been fully-rendered at build-time.
+When `edgeMiddleware` is enabled, an edge function will execute your middleware code for all requests including static assets, prerendered pages, and on-demand rendered pages.
+
+For on-demand rendered pages, the locals are serialized using JSON and sent in a header for the serverless function, which performs the rendering. As a security measure, the serverless function will refuse to serve requests with a `403 Forbidden` response unless they come from the generated edge function.
```

Comment:
```suggestion
For on-demand rendered pages, the data in `locals` is serialized using JSON and sent in a header for the serverless function, which performs the rendering. As a security measure, the serverless function will refuse to serve requests with a `403 Forbidden` response unless they come from the generated edge function.
```
Would this be a correct way to say it? Rather than saying "the locals", which sounds a bit odd to me?  (Again, whatever change you make here, make in the Vercel file too.)

Reply:
`context.locals` comes across as the clearest to me.